tests/unit/config/config_test.py

```diff
@@ -1080,8 +1097,8 @@ def test_resolve_environment_nonexistent_file(self):
                 {'foo': {'image': 'example', 'env_file': 'nonexistent.env'}},
                 working_dir='tests/fixtures/env'))
 
-            assert 'Couldn\'t find env file' in exc.exconly()
-            assert 'nonexistent.env' in exc.exconly()
+        assert 'Couldn\'t find env file' in exc.exconly()
+        assert 'nonexistent.env' in exc.exconly()
```

Comment:
These assertions weren't running because they were in the `with` block. I noticed this while looking around to make sure we had a test that covered the case where `env_file` was a string (this test covers it).